Source/DafnyLanguageServer.Test/Lookup/HoverVerificationTest.cs

```diff
@@ -223,12 +220,11 @@ method Abs(x: int) returns (y: int)
       );
     }
 
-    private async Task<TextDocumentItem> GetDocumentItem(string source, string filename, CompilationStatus expectedStatus) {
+    private async Task<TextDocumentItem> GetDocumentItem(string source, string filename) {
       source = source.TrimStart();
       var documentItem = CreateTestDocument(source, filename);
       await client.OpenDocumentAndWaitAsync(documentItem, CancellationToken);
-      var lastStatus = await WaitUntilDafnyFinishes(documentItem);
-      Assert.AreEqual(expectedStatus, lastStatus);
```

Comment:
I like that you have a new way more robust to get the last status; however, you removed one test here. Is there are good reason why you removed it?